next.config.js

```diff
@@ -27,4 +29,8 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+const sentryWebpackPluginOptions = {
+  silent: !!isProd,
```

Comment:
여기서 `!!`는 boolean 형을 보장하기 위해서 인가요 ??

그냥 `isProd`로 작성하지 않으신 이유가 궁금해요 !!

Reply:
맞아요! 근데 isProd가 true일때 silent 옵션이 false가 되야하는게 맞는거 같기도...

Reply:
그렇네요 ! isProd가 true일 때 true가 되는게 맞네요 !

---
boolean 형 보장을 위해서는 이런 방법은 어떨까요 ?
```suggestion
  silent: Boolean(isProd),
```

Reply:
오호 앞으로 참고하겠습니다. 더 보기도 편하네요!